proto/provenance/marker/v1/genesis.proto

```diff
@@ -19,4 +19,18 @@ message GenesisState {
 
   // A collection of marker accounts to create on start
   repeated MarkerAccount markers = 2 [(gogoproto.nullable) = false];
+
+  // list of denom based denied send addresses
+  repeated DenySendAddress deny_send_addresses = 4 [(gogoproto.nullable) = false];
 }
+
+// DenySendAddress defines addresses that are denied sends for marker denom
+message DenySendAddress {
+  option (gogoproto.equal)           = false;
+  option (gogoproto.goproto_getters) = false;
+
+  // marker_address is the marker's address for denied address
+  string marker_address = 1;
+  // deny_address defines all wallet addresses that are denied sends for the marker
+  string deny_address = 3;
```

Comment:
what happened to position 2 in this proto?

Reply:
Avoid merge conflict with Nav PR. This was on purpose.

Reply:
I don't see the update from the NAV PR which has been merged in...

Reply:
🤦 ugh, I had my wires crossed.  I did an increment jump in the GenesisState proto in prep for merge.  This one was a mistake.  Thought I was looking at the GenesisState here in the 4 lines the comment show.  Sorry.  Fixed.